packages/components/molecules/f-searchbox/src/components/Form.vue

```diff
@@ -328,8 +328,6 @@ export default {
 
             if (this.isValid === true) {
                 this.setErrors([]);
-                this.clearAddressValue(this.shouldClearAddressOnValidSubmit);
```

Comment:
Should we duplicate this before the [`return false;`](https://github.com/justeat/fozzie-components/pull/1258/files#diff-65aa0d48592087454609523d61236ae899bd7efda26d0f5caf5323837dac1745R340) line, so that this behaviour remains consistent? Or is that not necessary?

Reply:
yeah, good shout — have updated.